packages/halo-theme/src/shared-styles/scrollbar.less

```diff
@@ -1,5 +1,18 @@
 @import '@refinitiv-ui/elemental-theme/src/shared-styles/scrollbar';
 
+.standard-scrollbars() {
```

Comment:
let's apply the same change of `packages/elemental-theme/src/shared-styles/scrollbar.less` to here.